test/parallel/xyk-pallet.L1asset.register.update.test.ts

```diff
@@ -99,34 +98,20 @@ test("GIVEN Create one asset with the same address but different chains THEN Ope
 });
 
 test("GIVEN Create one asset with the same address and same chain THEN Operation fail", async () => {
-  const assetId1 = await api.query.tokens.nextCurrencyId();
   const tokenAddress = "0x" + randomBytes(20).toString("hex");
 
-  await sudo
-    .registerL1Asset(assetId1, tokenAddress, "Ethereum")
-    .then((result) => {
-      const eventResponse = getEventResultFromMangataTx(result);
-      expect(eventResponse.state).toEqual(ExtrinsicResult.ExtrinsicSuccess);
-    });
-
-  const assetId2 = await api.query.tokens.nextCurrencyId();
+  await sudo.registerL1Asset(null, tokenAddress, "Ethereum").then((result) => {
+    const eventResponse = getEventResultFromMangataTx(result);
+    expect(eventResponse.state).toEqual(ExtrinsicResult.ExtrinsicSuccess);
+  });
 
   await sudo
-    .registerL1Asset(assetId2, tokenAddress, "Ethereum")
+    .registerL1Asset(null, tokenAddress, "Ethereum")
     .then(async (result) => {
       const sudoEvent = await getEventErrorFromSudo(result);
       expect(sudoEvent.state).toBe(ExtrinsicResult.ExtrinsicFailed);
       expect(sudoEvent.data).toBe("ConflictingL1Asset");
     });
-
-  const idToL1Asset1 = JSON.parse(
-    JSON.stringify(await api.query.assetRegistry.idToL1Asset(assetId1)),
-  );
-  const idToL1Asset2 = JSON.parse(
-    JSON.stringify(await api.query.assetRegistry.idToL1Asset(assetId2)),
-  );
-  expect(idToL1Asset1.ethereum).toEqual(tokenAddress);
-  expect(idToL1Asset2).toEqual(null);
```

Comment:
why not using "l1AssetToId" and validate that the token only exists on Ethereum once? ( perhaps overkill ? )

Reply:
I'm unsure how we can check it because the second operation doesn't return any assetId1... 